test/unit/auth/token-verifier.spec.ts

```diff
@@ -30,10 +30,10 @@ import LegacyFirebaseTokenGenerator = require('firebase-token-generator');
 import * as mocks from '../../resources/mocks';
 import { FirebaseTokenGenerator, ServiceAccountSigner } from '../../../src/auth/token-generator';
 import * as verifier from '../../../src/utils/token-verifier';
-import * as verifierUtil from '../../../src/auth/token-verifier-util';
+import * as verifierAuth from '../../../src/auth/token-verifier';
```

Comment:
Easier and less ambiguous to just import named members here.

```
import { ID_TOKEN_INFO, SESSION_COOKIE_INFO } from '...';
```

Reply:
Updated!